repairnator/kubernetes-support/repairnator-deployment-yamlfiles/repairnator-worker.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: repairnator-worker
+  labels:
+    app: repairnator-worker
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: repairnator-worker
+  template:
+    metadata:
+      labels:
+        app: repairnator-worker
+    spec:
+      containers:
+      - name: repairnator-worker
+        image: repairnator/repainator-worker:latest
+        env: 
+        - name: MONGODB_HOST
+          value: mongodb://mongo:27017 # default if service of Mongodb deployed on K8s is called mongo with port 27017.
+        - name: MONGODB_NAME
+          value: admin   # Name of the database for the pipeline to store files when done repairing,
+                         # Also note that the db choosed need to have write access like admin default database
+        # Optional for pushing or creating PR to github repo provided with the PUSH_PR otherwise can be left blank
+        - name: GITHUB_OAUTH
+          value: 3aa2598e2070b94aae42274440c4f54fb720e499
```

Comment:
Remove all of your personal account information.

Reply:
Fixed.